ecl/hql/hqlgram.y

```diff
@@ -3768,6 +3768,12 @@ soapFlag
                             parser->normalizeExpression($3, type_string, false);
                             $$.setExpr(createExprAttribute(logAtom, $3.getExpr()), $1);
                         }
+    | TOK_LOG '(' expression ',' expression ')'
+                        {
+                            parser->normalizeExpression($3, type_string, false);
+                            parser->normalizeExpression($5, type_string, false);
+                            $$.setExpr(createExprAttribute(logAtom, createComma($3.getExpr(), $5.getExpr())), $1);
```

Comment:
minor: Normally you would not have a comma inside an attribute.  i.e:

    createExprAttribute(logAtom, $3.getExpr(), $5.getExpr())

and then use unwindChildren() to extract them

Reply:
Gotcha.  I think I remember how a comma might be used now, it's been a while.  I'll fix it.

Reply:
The commas are mainly used for datasets.  At some point I may get rid of most/all of them and use initializer lists instead.